onnxmltools/utils/float16_converter.py

```diff
@@ -13,21 +13,23 @@ def npfloat16_to_int(np_list):
     return [int(bin(_.view('H'))[2:].zfill(16), 2) for _ in np_list]
 
 
-def float16_converter(model_path):
+def float16_converter(onnx_file, new_onnx_file):
```

Comment:
Yes. Here is an example,
        onnx_file = 'c:/winmlperf_coreml_SqueezeNet_prerelease.onnx'
        new_onnx_file = 'c:/winmlperf_coreml_SqueezeNet_prerelease_float16.onnx'
        float16_converter(onnx_file, new_onnx_file)

Reply:
I see. Do you think `def float16_converter(onnx_model)` a better interface? Using files as function I/O may add some friction when user has already loaded the model.

Reply:
You mean use protobuf model rather than onnx file?

Reply:
Yes!

Reply:
The intention of this converter is to serve as a standalone feature.  That's why it consumes/generates onnx file. But it can be easily done if we want it to consume/generate protobuf model

Reply:
ok. I can change the input/output to onnx model. If there is any need to save/read model to file, a separate call to save_model()/load_model() can do